src/pages/learn.js

```diff
@@ -57,14 +58,17 @@ const callStitch = async (metadata, key, callback) => {
     callback(parseArticles(res));
 };
 
-export default () => {
+export default ({ location }) => {
     const metadata = useSiteMetadata();
     const [articles, setArticles] = useState([]);
-    const [filterValue, setFilterValue] = useState({});
+    const { search = {} } = location;
+    const [filterValue, setFilterValue] = useState(parseQueryString(search));
     useEffect(() => {
+        window.history.pushState({}, '', buildQueryString(filterValue));
```

Comment:
I spent a while trying to use a gatsby or reach-router function here instead but couldn't get it to work correctly. Let me know if you think this is an issue!

Reply:
I saw this example
```
import { navigate } from "gatsby";
    navigate("/app/login")
```

Would that work here?

Reply:
it does work but it changes the url so it brings you to the top of the page when it updates

Reply:
Ah, could we (just a thought), add an id to the filter bar and link to that with a `#`?